src/library_syscall.js

```diff
@@ -1175,6 +1183,14 @@ var SyscallsLibrary = {
     }
 #endif // SYSCALLS_REQUIRE_FILESYSTEM
   },
+  __syscall240: function(which, varargs) { // futex
+    return -{{{ cDefine('ENOSYS')}}};
+  },
```

Comment:
Same question for futex; would it make more sense to #ifdef out code that uses it?